### ISSUE TITLE
Command for flushing the history

### ISSUE THREAD

**Issue by ashishnegi** [mridgers/clink#499]
Hi
Thanks for this clink.

Is it possible to flush the clink history when command prompt is closing ? 
Otherwise, can we do it after say configurable number of commands ?

Thanks

**Comment by CollinChaffin** [comment]
Yep, just use it normally and I assure you that sooner rather than later if you have the setting set to write history to disk, you'll wind up with a sudden zero-byte .history file and lose ALL your history.  It happens to me constantly without a fix for years. :(
